war/src/main/less/modules/page-header.less

```diff
@@ -0,0 +1,226 @@
+@import "../abstracts/colors.less";
+
+.page-header {
+    display: block;
+    display: flex;
+    align-items: center;
+    height: 3.5rem;
+
+    font-family: Roboto, Helvetica, Arial, sans-serif;
+    font-size: 1rem;
+    line-height: 1.5;
+
+    background-color: @header-bg-classic;
+}
+
+.page-header > * {
+    margin-right: 0.75rem;
+}
+
+.page-header__brand {
+    display: inline-block;
+    height: 100%;
+    position: relative;
+    flex: 1; // push controls to the end of the block
+}
+
+// Need to use the element selector to increase weight otherwise it will be overriden by the
+// a:visited selector if it is declared later
+// Only styled by the overrides with the new UI enabled
+a.page-header__brand-link {
+    display: none;
+}
+
+.page-header__brand-name {
+    color: inherit;
+}
+
+.page-header__brand-image {
+    height: 32px;
+    height: 2rem;
```

Comment:
Same comment as above with https://caniuse.com/#search=rem

Reply:
The reason I did this for the logo is to have an extra piece of mind. Without this, in really old browsers the logo would go fullscreen. That way, you wouldn't even see a faulty page.